src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -625,7 +649,6 @@ internal void SetGroupInputPorts()
         /// </summary>
         internal void SetGroupOutPorts()
         {
-            OutPorts.Clear();
```

Comment:
why did you remove `OutPorts.Clear();` ?

Reply:
so now `SetGroupInputPorts` and `SetGroupOutPorts` will always append ports to `OutPorts` and `InPorts`.  I hope nobody uses these functions expecting the port arrays to be `re-set` ...as the func names suggest. Can you make these functions private and maybe add to the description that they always append to the respective port arrays ?

Reply:
Sure, editted.